lib/chat_api_web/controllers/customer_controller.ex

```diff
@@ -8,6 +8,25 @@ defmodule ChatApiWeb.CustomerController do
 
   action_fallback(ChatApiWeb.FallbackController)
 
+  plug :authorize when action in [:show, :update, :delete]
+
+  defp authorize(conn, _) do
+    id = conn.path_params["id"]
+
+    preloads =
+      (conn.path_params["expand"] || ["company", "tags"])
```

Comment:
hmm i wonder if we can remove the "company"/"tags" defaults... probably safer to keep them in for now 👍 (just commenting so i don't forget)

Reply:
It’s from the old `get_customer!/1` and removing this default seems to break three or four tests.

Reply:
makes sense!